### ISSUE TITLE
Should #nullable and #pragma nullable have effect on all warnings related to nullable feature rather than only on those reported by flow analysis?

### ISSUE THREAD

**Issue by AlekseyTs** [dotnet/roslyn#32742]
Right now ```#nullable``` and ```#pragma nullable``` have effect only on warnings that are reported by flow analysis. Other nullable related warnings, for example, for various mismatches in declarations are enabled by default. Should all those warnings be disabled by default and obey ```#nullable``` and ```#pragma nullable``` settings?

**Comment by jcouv** [comment]
Relates to https://github.com/dotnet/roslyn/issues/35748 (removing `#pragma warning ... nullable` and adding `#nullable ... warnings`)

**Comment by jcouv** [comment]
Assigned to @RikkiGibson Let's see if we can take this one past 7/16.
